include/quda.h

```diff
@@ -1280,6 +1287,35 @@ extern "C" {
    */
   void dslashQuda(void *h_out, void *h_in, QudaInvertParam *inv_param, QudaParity parity);
 
+  /**
+   * Apply the covariant derivative.
+   * @param h_out  Result spinor field
+   * @param h_in   Input spinor field
+   * @param dir    Direction of application
+   * @param param  Metadata for host and device storage
+   */
+  void covDevQuda(void *h_out, void *h_in, int dir, QudaInvertParam param);
```

Comment:
The last argument should be passed by address (value) rather than by value.

Reply:
fixed in 92e6bf8